### PR TITLE
Fixes to Latex table formatting

### PR DESCRIPTION
**Pull request by emmetfrancis**
Main changes to latex table formatting:
- Fix formatting of reaction equations - add brackets for subscripts {}, delete "*", replace "**" with "^"
- Fix printing of initial condition pint quantity (handle case in which the magnitude is a string)
- Make sure tables display index (names of each object) and ensure Latex table is printed instead of returning a pandas dataframe
- Print reaction topology

### CODE REVIEW

**Comment by emmetfrancis** [comment]
Let's merge this before merging the other PR if that sounds alright. This is just some extra fixes I made to get the Latex tables closer to correct formatting

File: smart/solvers.py
Comment (finsberg) [inline smart/solvers.py]: Perhaps we can leave this part out of this PR, since it is already covered by https://github.com/RangamaniLabUCSD/smart/pull/163?

Comment (emmetfrancis) [reply smart/solvers.py]: Great catch, done!

**Review by finsberg — APPROVED** [review]

**Pull request merged**
